_entities.md

```diff
@@ -416,6 +434,14 @@ type | The type of endpoint. Possible values are 'email', 'card' and 'external'.
 
 The `user` object contains all information we have on record relating to the currently logged in user.
 
+<aside class="warning">
```

Comment:
We will handle this particular communication through a different channel, you can remove this section. /cc @ruimarinho